db/bigtable.go

```diff
@@ -560,32 +693,49 @@ func (bigtable *Bigtable) SaveAttestations(blocks map[uint64]map[string]*types.B
 
 	for _, slot := range slots {
 		for _, b := range blocks[slot] {
-			logger.Infof("processing slot %v", slot)
-			for _, a := range b.Attestations {
-				for _, validator := range a.Attesters {
-					inclusionSlot := slot
-					attestedSlot := a.Data.Slot
-					if attestationsBySlot[attestedSlot] == nil {
-						attestationsBySlot[attestedSlot] = make(map[uint64]uint64)
-					}
+			if b.Status == 2 {
```

Comment:
please create a named constant for 2

Reply:
will be done in a separate PR to not further bloat this one